ruby-challenges.rb

```diff
@@ -17,7 +25,46 @@
 us_states = { northwest: ['Washington', 'Oregon', 'Idaho'], southwest: ['California', 'Arizona', 'Nevada'], notheast: ['Maine', 'New Hampshire', 'Rhode Island'] }
 # Expected output: ['Arizona', 'California', 'Idaho', 'Maine', 'Nevada', 'New Hampshire', 'Oregon', 'Rhode Island', 'Washington'] 
 
+p us_states.values_at(:notheast, :northwest, :southwest).flatten.sort
```

Comment:
Logic looks great, just make sure to pay attention to if the prompt asks for a method.